### ISSUE TITLE
Problem with parsing

### ISSUE THREAD

**Issue by faridasabry** [citp/BlockSci#265]
I have downloaded the bitcoin blockchain on one machine, then I moved the /data/bitcoin-blockchain/blocks folder to another machine that meets the memory requirements for BlockSci

I installed it as per the steps in the compiling instructions section, I don't recall the messages appeared after installation but trying to run some script I found that only data for one block is shown. I re-run the parser to find the output below, I tried removing blocksci and all its directories reinstalled it and started the parser to receive the same messages below  for only the first block copied. so are their any other directories or files needed to be copied from the bitcoin blockchain?   (The blocks folder has all the blk*.dat and rev*.dat files and index folder copied as is from the machine running a full node)
Trying the following command:
`sudo blocksci_parser -o /data/blocksci-bitcoindata  core-update  --max-block 558109 disk -c /data/bitcoin-blockchain
`
I receive the result below after it took a long time fetching the block headers:
```
100.0% done fetching block headers
Starting with chain of 1 blocks
Removing 0 blocks
Adding 0 blocks
Updating hash index
Updating index with 1 txes
0.00% done
Updating address index
Updating index with 1 txes
0.00% done
Updating index with scripts of type multisig_script
```

**Comment by maltemoeser** [comment]
Hm this is strange. The block files are in the folder `/data/bitcoin-blockchain/blocks/`? Why are you using sudo?

**Comment by faridasabry** [comment]
Yes /data/bitcoin-blockchain contains all files from .bitcoin, it was missing the chain state folder the last time (don't know why) but after syncing the folder again is the same as  .bitcoin from Bitcoin Core.
I repeated all the installation steps again, now I am receiving this error:
terminate called after throwing an instance of 'std::bad_alloc'
  what():  std::bad_alloc
Aborted (core dumped)

I checked this issue # [39](https://github.com/citp/BlockSci/issues/39) but I couldn't find this blockList.dat file mentioned to be removed.
The machine also has enough memory:
`free -g
              total        used        free      shared  buff/cache   available
Mem:            125           1           4           0         119         122
Swap:            22           0          22`

**Comment by maltemoeser** [comment]
Make sure to delete the contents of the output directory before you attempt to reparse.

If you can, maybe sync a few blocks with Bitcoin Core on the machine you've installed BlockSci on to see if it works with its block files.

**Comment by faridasabry** [comment]
It worked so it seems it depends on some other files/information from BitcoinCore files, why there is no debug option in  blocksci_parser

I tried this:
`blocksci_parser --output-directory blocksci-parsing update --max-block 10000 disk --coin-directory .bitcoin`

The output was:
```
100.0% done fetching block headers
Starting with chain of 0 blocks
Removing 0 blocks
Adding 10000 blocks
99.09% done, Block 9911/10000
Back linking transactions

Updating hash index
Updating index with 10092 txes
99.09% done
Updating address index
Updating index with 10092 txes
99.09% done
Updating index with scripts of type multisig_script
```
From the output, it looks that there are some missing blocks. However, when I tried the blocksci notebook demo part below, it showed all the 10000 blocks.
`tx_size = [(block.height, np.mean(block.txes.size_bytes)) for block in chain]
df_tx_size = pd.DataFrame(tx_size, columns=["Height", "Avg. tx size"])
df = chain.heights_to_dates(df_tx_size)
print(df)
`
